java/pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>ola</groupId>
   <artifactId>ola-java-client</artifactId>
-  <version>0.1.0</version>
+  <version>0.0.3</version>
```

Comment:
We should never go backwards in our versions, even if I failed to update the separate Java changelog last time around.

I think this should be 0.2.0 given:
> MINOR version when you add functionality in a backward compatible manner

Reply:
Fixed with commit 79630929f9cde69e0dbff67f7e1cbdfa4e3acbb6